transforms/images/image-assembler-plugin/.bumpversion.cfg

```diff
@@ -0,0 +1,23 @@
+[bumpversion]
+current_version = 1.2.0-dev0
```

Comment:
We should remove the dev tag before the final merge.

Reply:
So I would imagine that this is part of the final review process, correct?